.idea/.idea.course-1.dir/.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+﻿# Default ignored files
```

Comment:
Would you mind removing this file from the PR and adding an `.idea/` to the main `.gitignore`?

Reply:
I did it can you check please?

Reply:
Yep, this looks great - thanks!